go-controller/pkg/factory/factory.go

```diff
@@ -670,10 +670,7 @@ func (wf *WatchFactory) addHandler(objType reflect.Type, namespace string, lsel
 }
 
 func (wf *WatchFactory) removeHandler(objType reflect.Type, handler *Handler) error {
-	if inf, ok := wf.informers[objType]; ok {
-		return inf.removeHandler(handler)
-	}
-	return fmt.Errorf("tried to remove unknown object type %v event handler", objType)
+	return wf.informers[objType].removeHandler(handler)
```

Comment:
if `informer[objType]` does not exist, this will panic, no?
should we keep the check and don't return anything if there is no an ` `informer[objType]``?